### ISSUE TITLE
specific field in tt_content not working

### ISSUE THREAD

**Issue by Finulin** [beechit/default_upload_folder#11]
default_upload_folders {

    # this works
    tt_content = 2:my_path

    # this works not
    tt_content.myimage = 2:my_path

    # core fields also not working
    tt_content.image = 2:my_path

}

I use default_upload_folder v2.0.1 on TYPO3 10

Thanks for your help

**Comment by SventB** [comment]
There's a bug in this extension. In file "DefaultUploadFolder.php" there's a "." (point) missing after $table:

```
        $subFolder = $defaultPageTs['default_upload_folders.'][$table.'.'][$field] ?? ''; 
        if (empty($subFolder)) {
            $subFolder = $userTsConfig['default_upload_folders.'][$table.'.'][$field] ?? '';
        }
        return $subFolder;
```

**Comment by Udo771** [comment]
@SventB thank you, this fixed the issue

**Comment by frans-beech-it** [comment]
Fixed with #20
